boa_engine/src/tests.rs

```diff
@@ -442,6 +442,15 @@ fn test_invalid_break_target() {
     let _ = &exec(src);
 }
 
+#[test]
+fn test_invalid_break() {
+    let src = r#"
+        break;
+        "#;
+
+    assert!(matches!(Context::default().eval(src.as_bytes()), Err(_)));
```

Comment:
Here too, I would use `is_err()` instead of `matches!()`, it makes things more clear, and you can directly pass a reference to `src` instead of calling `as_bytes()`, if I remember correctly.